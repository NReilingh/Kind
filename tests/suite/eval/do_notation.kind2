Main : Maybe U60
Main =
  do Maybe {
    ask a = Maybe.pure 2
    ask b = Maybe.pure 3
    Maybe.some (+ a b)
  }