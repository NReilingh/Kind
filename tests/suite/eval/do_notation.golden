(Maybe.some _ 5)

